app/windows/rule_editor_panel.py

```diff
@@ -19,6 +18,7 @@
     QVBoxLayout,
     QWidget,
 )
+from loguru import logger
```

Comment:
Same as above. Import groups should be sorted alphabetically.

Reply:
I just Used Optimize Imports feature in PyCharm and it arranged it